index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <!--link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css"-->
+        <link rel="stylesheet" type="text/css" href="http://yui.yahooapis.com/3.18.1/build/cssnormalize-context/cssnormalize-context-min.css">            
+        <link rel="stylesheet" href="style.css" />
+        <title>canvas</title>
+    </head>    
+    <body>
+ 	    <div class="wrapper">
+            <header class="block-header">
+                <div class="block_nav">
+                    <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd"
+                              d="M11 26H29V24H11V26ZM11 21H29V19H11V21ZM11 14V16H29V14H11Z"
+                              fill="black" fill-opacity="0.54"/>
+                    </svg>
+                </div>
+                <div class="block-logo">
+                    <p>CodeJam - Canvas</p>
+                </div>
+                <div class="block_menu">
+                    <svg width="12" height="24" viewBox="0 0 12 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd"
+                              d="M6 8C7.1 8 8 7.1 8 6C8 4.9 7.1 4 6 4C4.9 4 4 4.9 4 6C4
+                              7.1 4.9 8 6 8ZM6 10C4.9 10 4 10.9 4 12C4 13.1 4.9 14 6 14C7.1
+                              14 8 13.1 8 12C8 10.9 7.1 10 6 10ZM6 16C4.9 16 4 16.9 4 18C4
+                              19.1 4.9 20 6 20C7.1 20 8 19.1 8 18C8 16.9 7.1 16 6 16Z"
+                              fill="black" fill-opacity="0.54"/>
+                    </svg>
+                </div>
+            </header>
+
+            <main>
+                <div class="block-instruments">
+                    <div class="block-instruments__first">
+                        <img src="assets/Bottom-List-Instruments.png" alt="instruments">
+                    </div>
+                    <div class="block-instruments__last">
+                        <img src="assets/Bottom-List-Colors.png" alt="colors">
+                    </div>
+                </div>
+                <div class="block-canvas">
+					<canvas id="canvas" class="canvas" width="512" height="512">
+					</canvas>
+	            </div>
+                <div class="block-buttons">
+                    <input id="4x4" class="button4x4" type="submit" value="4 x 4">
```

Comment:
You don't submit anything here, so <input type="button"> or <button> would be better here